components/msg/constellation_msg.rs

```diff
@@ -162,6 +163,15 @@ pub enum TraversalDirection {
     Back(usize),
 }
 
+pub struct HistoryEntry {
+    pub url: ServoUrl
```

Comment:
Title and favicon URL. I'll add these if you agree that the proposed approach make sense.